tools/korebuild.schema.json

```diff
@@ -109,9 +109,9 @@
     "channel": {
       "description": "The channel of KoreBuild used to select a version when korebuild-lock.txt is not present.",
       "type": "string",
-      "default": "master",
+      "default": "main",
       "enum": [
-        "master",
+        "main",
         "release/2.2",
         "release/2.1",
         "release/2.0"
```

Comment:
These channels all still exist. Removing the `master` channel is fine because nobody will change the URIs in existing korebuild.json files.